web/src/components/AppBar/AppBar.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-app-bar
+    app
+  >
+    <v-toolbar-title>
+      <v-img
+        alt="DANDI logo"
+        contain
+        max-height="48px"
+        max-width="120px"
+        src="@/assets/logo.svg"
+      />
+    </v-toolbar-title>
+
+    <v-btn
+      :to="{ name: 'home' }"
+      exact
+      text
+    >
+      Welcome
+    </v-btn>
+    <v-btn
+      :to="{ name: 'publicDandisets' }"
+      exact
+      text
+    >
+      Public Dandisets
+    </v-btn>
+    <v-btn
+      v-if="loggedIn"
+      :to="{ name: 'myDandisets' }"
+      exact
+      text
+    >
+      My Dandisets
+    </v-btn>
+    <v-btn
+      :href="dandiUrl"
+      target="_blank"
+      rel="noopener"
+      text
+    >
+      About
+      <v-icon>mdi-open-in-new</v-icon>
+    </v-btn>
+
+    <v-spacer />
+
+    <span
+      v-if="!loggedIn"
+    >
+      Want to create your own datasets?
+    </span>
+    <v-btn
+      v-if="!loggedIn"
+      :to="{ name: 'userRegister' }"
+      color="primary"
+      outlined
+      rounded
+    >
+      Create Account
+    </v-btn>
+    <v-btn
+      v-if="!loggedIn"
+      :to="{ name: 'userLogin' }"
+      color="primary"
+      rounded
+    >
+      Login
+    </v-btn>
+    <UserMenu
+      v-if="loggedIn"
+    />
```

Comment:
To avoid duplicate v-ifs on the same condition:

```suggestion
    <UserMenu
      v-if="loggedIn"
    />
    <template v-else>
      <span>Want to create your own datasets?</span>
      <v-btn
        :to="{ name: 'userRegister' }"
        color="primary"
        outlined
        rounded
      >
        Create Account
      </v-btn>
      <v-btn
        :to="{ name: 'userLogin' }"
        color="primary"
        rounded
      >
        Login
      </v-btn>
    </template>
```

Reply:
I considered that, but figured we may want to tweak the conditional visibility of some of these things (particularly the "Want to create your own datasets?" element). However, for "Create Account" and "Login", they'll always be visible under the same circumstances, so it may be easier to comprehend if we group them together in a `template` block.

I'l make that change.

Reply:
Fixed this, but left the "Want to create your own datasets?" out for now, because it may change and is easier to tweak in development if it's separate.